packages/manager/cypress/e2e/core/images/create-image.spec.ts

```diff
@@ -84,4 +120,136 @@ describe('create image (e2e)', () => {
         });
     });
   });
+
+  it('displays notice informing user that Images are not encrypted, provided the LDE feature is enabled and the selected linode is not in an Edge region', () => {
+    // Mock feature flag -- @TODO LDE: Remove feature flag once LDE is fully rolled out
+    mockAppendFeatureFlags({
+      linodeDiskEncryption: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Mock responses
+    const mockAccount = accountFactory.build({
+      capabilities: ['Linodes', 'Disk Encryption'],
+    });
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetRegions(mockRegions).as('getRegions');
+    mockGetLinodes(mockLinodes).as('getLinodes');
+
+    // intercept request
+    cy.visitWithLogin('/images/create');
+    cy.wait([
+      '@getFeatureFlags',
+      '@getClientStream',
+      '@getAccount',
+      '@getLinodes',
+      '@getRegions',
+    ]);
+
+    // Find the Linode select and open it
+    cy.findByLabelText('Linode')
+      .should('be.visible')
+      .should('be.enabled')
+      .should('have.attr', 'placeholder', 'Select a Linode')
+      .click();
+
+    // Select the Linode
+    ui.autocompletePopper
+      .findByTitle(mockLinodes[0].label)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    // Check if notice is visible
+    cy.get(`[data-testid=${encryptionCaveatNoticeTestId}]`).should('exist');
```

Comment:
```suggestion
    cy.get(`[data-testid=${encryptionCaveatNoticeTestId}]`).should('be.visible');
```
Just a small suggestion, since this will also catch cases where the element is hidden (e.g. by a CSS rule) or obscured by another element